packages/web/src/components/SongsCard/SongsCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './styles.scss';
+import QuickMenu from './QuickMenu';
+import useQuickMenu from '../../custom-hooks/useQuickMenu';
+
+function SongsCard() {
+    const [open, id, cardId, openMenu] = useQuickMenu();
+
+    return (
+        <div className="songsCard">
+            <input
+                className="songsCard__3pointButton"
+                type="button"
+                onMouseDown={openMenu}
+                value="OPTIONS"
+            />
+            {open && id === cardId && <QuickMenu />}
```

Comment:
render with two conditions:
1. if open is true
2. if redux id its the same as local card id

without the second condition redux triggers all the song cards quick menus.